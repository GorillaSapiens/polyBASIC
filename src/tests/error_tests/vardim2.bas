DIM a(5,8)
a = 2
