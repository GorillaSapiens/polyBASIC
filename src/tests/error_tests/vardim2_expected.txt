ERROR: @2:12, ARRAY USED AS VARIABLE ❰a❱ @1
